plugins/analogix/fu-analogix-device.c

```diff
@@ -190,12 +185,12 @@ fu_analogix_device_setup (FuDevice *device, GError **error)
 		return FALSE;
 
 	/*  get custom version */
-	if (!fu_analogix_device_receive (self, ANX_BB_RQT_READ_CUS_VER, 0, 0,
-					 &buf_custom[1], 1, error))
-		return FALSE;
-	if (!fu_analogix_device_receive (self, ANX_BB_RQT_READ_CUS_RVER, 0, 0,
-				         &buf_custom[0], 1, error))
-		return FALSE;
+//	if (!fu_analogix_device_receive (self, ANX_BB_RQT_READ_CUS_VER, 0, 0,
+//					 &buf_custom[1], 1, error))
+//		return FALSE;
+//	if (!fu_analogix_device_receive (self, ANX_BB_RQT_READ_CUS_RVER, 0, 0,
+//				         &buf_custom[0], 1, error))
+//		return FALSE;
 	self->fw_version = fu_common_read_uint16 (buf_fw, G_LITTLE_ENDIAN);
```

Comment:
@LuckyCandice can you add the ANX_BB_RQT_READ_CUS_VER and ANX_BB_RQT_READ_CUS_RVER to this branch too please -- then we can drop the commit that disables these

Reply:
i.e. I think the` #define ANX_BB_RQT_READ_CUS_RVER 0xSOMETHIN`G is missing

Reply:
Done.

Reply:
Great, thanks. I've reverted the commit that disables these.